platform/engineering/frontend/eslint-additional-rules.md

```diff
@@ -0,0 +1,73 @@
+# ESLint Additional Rules
+
+The front-end team will be introducing a new set of rules from a third-party plugin called SonarJS ([eslint-plugin-sonarjs](https://github.com/SonarSource/eslint-plugin-sonarjs)).
```

Comment:
The purpose of release notes should be to document updates to the eslint rules and new rules that are under review. The overview should focus on explaining the review process or linking to a document that explains this process. I think the sonarjs stuff is good to add to another type of "roadmap" document that outlines the rulesets we intend to evaluate in the future. We could link to that in the overview too. 

The rest of the document should be structured: 

- with the latest release is at the top 
- so that each release has a section for 
  - New CI blocking rules with the release date (we can play with this wording / nothing released this sprint) 
  - Rules under review and their intended release date 

Some examples of good release notes are [here](https://www.prodpad.com/blog/writing-release-notes/).